testgrid/cmd/issue_gatherer/issuescraper.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue_gatherer
```

Comment:
a) Underscores seem to violates style -- both golang style and the style of this repo. Can you come up with a better package name?

b) Given that this is testgrid/cmd/issue_gatherer I would expect this to be `package main`